easybuild/easyblocks/l/lammps.py

```diff
@@ -272,12 +431,16 @@ def sanity_check_step(self, *args, **kwargs):
         if self.toolchain.options.get('usempi', None):
             custom_commands = [self.toolchain.mpi_cmd_for(cmd, 1) for cmd in custom_commands]
 
+        # Requires liblammps.so to be findable by the runtime linker (which it might not be if using
+        # rpath and filtering out LD_LIBRARY_PATH)
```

Comment:
@boegel This is an interesting potential issue for EESSI

Reply:
So, can we:
* skip this sanity check if rpath is enabled?
* proceed to merge and follow on with a fix for this later?
* some other option?

I am keen to get this PR merged...

Reply:
The fix is included here (leveraging `LIBRARY_PATH`), works for EESSI (which is all I really care about right now...)

Reply:
Ah, sorry. I misunderstood your reason for pointing this out to @boegel